pkg/util/util.go

```diff
@@ -69,6 +69,14 @@ func logPanic(r interface{}) {
 	glog.Errorf("Recovered from panic: %#v (%v)\n%v", r, r, callers)
 }
 
+// Min will return the smaller of the 2 given integers.
```

Comment:
sure. Though it's getting harder to ignore the similarity in the if/else clause in `manageReplicas`.